spring-cloud-gcp-starters/spring-cloud-gcp-starter-storage/pom.xml

```diff
@@ -26,5 +26,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-storage</artifactId>
 		</dependency>
```

Comment:
@artembilan I don't understand why do we need the web dependency here, was it just for a test as a resource server? Was it the only way to test? I don't oppose this, I just find it a bit peculiar the need for a servlet engine just for testing.